eclair-node/src/main/scala/fr/acinq/eclair/payment/LocalPaymentHandler.scala

```diff
@@ -27,34 +27,39 @@ class LocalPaymentHandler(nodeParams: NodeParams) extends Actor with ActorLoggin
 
   // TODO: store this map on file ?
   // TODO: add payment amount to the map: we need to be able to check that the amount matches what we expected
```

Comment:
~~// TODO: add payment amount to the map: we need to be able to check that the amount matches what we expected~~